buildenv/jenkins/common/pipeline-functions

```diff
@@ -143,11 +142,19 @@ def build_with_slack(JOB_NAME, PARAMETERS) {
         } catch (err) {
             echo "Couldn't retrieve downstream failed job url"
         }
-        MESSAGE = "Downstream job ${JOB_NAME} did not pass. Job Number: ${DOWNSTREAM_JOB_NUMBER} Job URL: ${DOWNSTREAM_JOB_URL}"
-        if (SLACK_CHANNEL) {
-            slackSend channel: SLACK_CHANNEL, color: 'danger', message: "Failure in: ${env.JOB_NAME} #${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)\nDownstream Job: ${JOB_NAME} #${DOWNSTREAM_JOB_NUMBER} (<${DOWNSTREAM_JOB_URL}|Open>)"
+        
+        if (JOB.result == "FAILURE"){
+        	if (SLACK_CHANNEL) {
+        		slackSend channel: SLACK_CHANNEL, color: 'danger', message: "Failure in: ${env.JOB_NAME} #${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)\nDownstream Job: ${JOB_NAME} #${DOWNSTREAM_JOB_NUMBER} (<${DOWNSTREAM_JOB_URL}|Open>)"
+        	}
+        	error "Downstream job ${JOB_NAME} did not pass. Job Number: ${DOWNSTREAM_JOB_NUMBER} Job URL: ${DOWNSTREAM_JOB_URL}"
+        } else if (JOB.result == "UNSTABLE") {
+        	echo "WARNING: ${env.JOB_NAME} ${env.BUILD_NUMBER} is unstable"
```

Comment:
This warning should mention the downstream job name and number, similar to the error case.